### ISSUE TITLE
Clicking on the version number facet does not copy version

### ISSUE THREAD

**Issue by mpmxyz** [Yellow-Dog-Man/Resonite-Issues#1665]
### Describe the bug?

The click visuals work but in the log file it only logs "Copying text of length 18 to clipboard" without a matching "Finished copying text of length 18 to clipboard".
The OS clipboard has not been touched.

### To Reproduce

Note: I had trouble reproducing the issue. So far I don't know a sure way but I observed a possible relation to an error popup on Resonite start and it being the first time Resonite has been started after booting Windows.

SystemHelper seems to be involved. Here is an approximate translation of the attached screenshot:
```
Error in System Helper Server: System.Net.Sockets.SocketException
(0x80004005): Could not establish connection because the target computer rejected the connection 127.0.0.1:62264
  at ...
```

Based on this I suspect the way to reproduce it is:
1. Boot PC
1. Start Resonite -> observe attached error popup
1. Click the version number facet

### Expected behavior

The version number text should now be in the clipboard.

### Screenshots

This is the observed error on Resonite startup:
![grafik](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/8538547/3ecbd194-14c2-4251-842b-04882ee52f17)

### Resonite Version Number

2024.4.11.484

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Desktop

### Log Files

[ANONYMOUS - 2024.4.11.484 - 2024-04-11 21_32_10.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/14951197/ANONYMOUS.-.2024.4.11.484.-.2024-04-11.21_32_10.log)

### Additional Context

I did not observe this error before testing the prerelease.

**Comment by Frooxius** [comment]
Do any other clipboard actions work at all?

From the look of it, this breaks the SystemHelper, which means no clipboard actions should work at all (since it's responsible for these), rather than just specifically the version number facet. 

Does this happen always? Do you have any firewalls or anything that would block connections between Resonite and SystemHelper?

**Comment by mpmxyz** [comment]
I haven't configured anything fancy with my firewall. My AVG antivirus reports that this module is not installed. So it should theoretically be Windows defaults.
I also had more trouble replicating it. I now checked my logfiles with a swift `find . -type f -print0 | xargs -0 grep "at SystemHelperClient.SystemHelper.GetUID ()"` found the following results:
```
./ANONYMOUS - 2024.4.11.484 - 2024-04-11 21_32_10.log:  at SystemHelperClient.SystemHelper.GetUID () [0x00011] in <173c45748316404eabad7e6707a74715>:0
./ANONYMOUS - 2024.4.3.1385 - 2024-04-04 21_33_55.log:  at SystemHelperClient.SystemHelper.GetUID () [0x00011] in <82f22875728649b6bb7fba25b4ea5efe>:0
./ANONYMOUS - 2024.4.5.1261 - 2024-04-06 15_42_01.log:  at SystemHelperClient.SystemHelper.GetUID () [0x00011] in <e653a61ae215412dadfd1c3c1b015d68>:0
./ANONYMOUS - 2024.4.9.1426 - 2024-04-10 18_05_17.log:  at SystemHelperClient.SystemHelper.GetUID () [0x00011] in <a0a146b3d73841d4b440908a6d895a1f>:0
```
That is not a lot but it always got me when I just found a bug and wanted to report it.
The issue seems to be triggered only a single time after the prerelease had been updated and (!) the PC had been rebooted.
Theories:
- The anti-virus checks files only once and only when they have been changed. The check does something wacky or it triggers some timeout or other time-related errors.
- Some kind of caching behaviour is triggered after a Resonite update and reboot which triggers some strange behaviour.

Since there is not enough data I'd recommend to close this issue until we have more info.
I'll keep a lookout for the popup and will look into the SystemHelper's log when it appears. (It is overwritten on every startup so I couldn't check now.)
